rafthttp/peer.go

```diff
@@ -161,8 +162,7 @@ func startPeer(tr http.RoundTripper, urls types.URLs, local, to, cid types.ID, r
 					if isMsgSnap(m) {
 						p.r.ReportSnapshot(m.To, raft.SnapshotFailure)
 					}
-					// TODO: log start and end of message dropping
-					plog.Warningf("dropping %s to %s since %s's sending buffer is full", m.Type, p.id, name)
+					status.dropMessage(m, fmt.Sprintf("sending buffer on %s is full", name))
```

Comment:
If we know the connection is inactive, then we can only print it in debug mode. 

If not, we should print out warn. Do we really need a new dropMessage func?